examples/local/101_initial_cluster.sh

```diff
@@ -18,6 +18,7 @@
 # required for a single shard deployment.
 
 source ./env.sh
+vtctld_grpc_port=15999
```

Comment:
This change can also be removed, since we aren't using this variable anywhere now